test/loadfile.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2023 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"sigs.k8s.io/yaml"
+)
+
+// loadYAML loads yaml from file to obj.
+func loadYAML(file string, obj interface{}) (err error) {
+	var data []byte
+	if data, err = ioutil.ReadFile(file); err != nil {
+		return
+	}
+	err = yaml.Unmarshal(data, obj)
+	return
+}
+
+// MustLoadYAML loads yaml or panics if the parse fails.
+func MustLoadYAML(file string, obj interface{}) {
+	err := loadYAML(file, obj)
+	if err != nil {
+		panic(fmt.Sprintf("load yaml file failed, file path: %s, err: %s", file, err))
+	}
+}
+
+// LoadJSON loads json from file to obj.
+func LoadJSON(file string, obj interface{}) (err error) {
```

Comment:
are LoadJSON and MustLoadJSON used? If not, I think there's no need to add them.